lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -171,8 +169,8 @@ data Depth = RootK | PurposeK | CoinTypeK | AccountK | RoleK | AddressK
 --
 -- FIXME: rename this to 'Role' or 'HDRole'
 data AccountingStyle
-    = UTxOExternal
-    | UTxOInternal
+    = UtxoExternal
+    | UtxoInternal
```

Comment:
I think this could be fixed in `fromTextToBoundedEnum SnakeLowerCase`.
It's spelt UTxO everywhere else.
This important word deserves special case treatment, so that we don't need to mangle our type names just to get the preferred serialisation.

Reply:
I thought of both option and should that it was cleaner in the end to rename the type here. It's perhaps a little less pervasive as a change to just deal with it in the toText//fromText instances. Any way requires a database fix / migration / work-around :|